app/config.go

```diff
@@ -0,0 +1,58 @@
+package app
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pelletier/go-toml/v2"
+
+	"github.com/jorgerojas26/lazysql/models"
+)
+
+type Config struct {
+	AppConfig   *models.AppConfig   `toml:"application"`
+	Connections []models.Connection `toml:"database"`
+}
+
+func defaultConfig() *Config {
+	return &Config{
+		AppConfig: &models.AppConfig{
+			DefaultPageSize: 300,
+		},
+	}
+}
+
+func LoadConfig() error {
+	configDir, err := os.UserConfigDir()
+	if err != nil {
+		return err
+	}
+	file, err := os.ReadFile(filepath.Join(configDir, "lazysql", "config.toml"))
```

Comment:
You should have an unexported method to compute the config file path instead of computing it twice

Reply:
I guess you mean:

> Would probably make sense to add an unexported method to compute the config file path instead of computing it twice?

Sure why not 😉